scripts/src/main/resources/scripts/command/android-studio

```diff
@@ -109,7 +109,13 @@ function doConfigureAndroidStudio() {
   fi
   if [ -n "${mode}" ]
   then
-    doConfigureWorkspace "${SETTINGS_PATH}/android-studio/workspace" "${WORKSPACE_PATH}" "${mode}"
+    local settings_workspace="${SETTINGS_PATH}/android-studio/workspace"
+    if [ -d "${settings_workspace}" ]
+    then
+      doConfigureWorkspace "${settings_workspace}" "${WORKSPACE_PATH}" "${mode}"
+    else
+      doWarning "There is no workspace folder in ${settings_workspace}. Launch Android Studio with default settings..."
```

Comment:
In case the folder `settings/android-studio/workspace` is deleted after the first launch of android studio, the tool will be started with settings defined in `workspaces/.../.android-studio` folder, if it is exists. So the the warning that android-studio will launch with default settings is not correct, is it? Should we just change the warning or add another check?

Reply:
I think this is fine. It is an odd edge-case. Technically what we want to say is that we do not manipulate/manage the settings and that is correct. If you want you can change the wording of the message but we should not change the logic of the code.